wallet/account.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package wallet
+
+import (
+	"crypto"
+
+	ed "github.com/oasisprotocol/curve25519-voi/primitives/ed25519"
+	"perun.network/go-perun/wallet"
+)
+
+// Account is an ed25519 signing key. It signs messages for a perun off-chain
+// identity in internet computer channels.
+type Account ed.PrivateKey
+
+var _ wallet.Account = (*Account)(nil)
+
+func (a Account) Address() wallet.Address {
```

Comment:
Please install a linter in the CI, as in our other go repositories.

Please document all Public types and functions.